Libraries/Vibration/Vibration.js

```diff
@@ -27,11 +27,31 @@ var Platform = require('Platform');
  */
 
 var Vibration = {
-  vibrate: function(duration: number = 400) {
+  vibrate: function(pattern: number | Array<number> = 400, repeat: number = -1) {
```

Comment:
@skv-headless I think it should be okay. Let's keep the Java code as it is, and change the JS to pass `-1` or `> -1`. So that if we need it, we can change it in future.